pkg/cmd/pr/merge/merge.go

```diff
@@ -131,6 +132,19 @@ func mergeRun(opts *MergeOptions) error {
 		return err
 	}
 
+	isTerminal := opts.IO.IsStdoutTTY()
+
+	if opts.AutoMergeDisable {
```

Comment:
It seems like enabling automerge should also short circuit the flow of the command. What was behind the decision to move that functionality down into the `mergePullRequest` function?

Reply:
@samcoe Since enabling auto-merges accepts identical inputs as the `mergePullRequest` mutation, I've decided to keep all behaviors of normal `pr merge` intact in `--auto` mode, including the interactive ones.